src/common/camera_control.h

```diff
@@ -22,6 +22,12 @@
 #include "common/darktable.h"
 
 #include <glib.h>
+
+#if defined (_WIN32)
```

Comment:
I have used only `_WIN32` , the others were already there. Agree that it should be unified, and I believe the `_WIN32` is the good, based on this article:
[_WIN32 or WIN32](http://stackoverflow.com/questions/662084/whats-the-difference-between-the-win32-and-win32-defines-in-c)